tests/__tests__/transpileIfTypescript.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { transpileIfTypescript } from '../../src';
+
+describe.only('transpileIfTypescript', () => {
+    it('should ignore anything non-TS', () => {
+        const contents = 'unaltered';
+        expect(transpileIfTypescript('some.js', contents)).toBe(contents);
+    });
+    it('should be able to transpile some TS', () => {
+        const ts = 'const x:string = "anything";';
+        expect(transpileIfTypescript('some.ts', ts)).toMatch('var x = "anything";');
+        expect(transpileIfTypescript('some.tsx', ts)).toMatch('var x = "anything";');
+    });
+
+    it('should be possible to pass a custom config', () => {
+        const customTsConfigFile = 'not-existant.json';
+        const customConfig = { __TS_CONFIG__: customTsConfigFile};
+        expect(() => transpileIfTypescript('some.ts', '', customConfig)).toThrow(new RegExp(customTsConfigFile));
+    });
+});
```

Comment:
2 small nits:

- Add a line at the end
- Rename the file in kebab case for consistency (the rest of the test files are named that way)